packages/mdc-textfield/README.md

```diff
@@ -329,102 +329,155 @@ Mixin | Description
+### MDCTextField
+#### Methods
 
-Property | Value Type | Description
---- | --- | ---
-`value` | `string` | Proxies to the foundation's `getValue`/`setValue` methods.
-`disabled` | `boolean` | Proxies to the foundation's `isDisabled`/`setDisabled` methods.
-`useNativeValidation` | `boolean` (write-only) | Proxies to the foundation's `setUseNativeValidation` method.
-`valid` | `boolean` | Proxies to the foundation's `isValid`/`setValid` methods.
-`helperTextContent` | `string` (write-only)| Proxies to the foundation's `setHelperTextContent` method when set.
-`ripple` | `MDCRipple` (write-only) | The `MDCRipple` instance for the root element that `MDCTextField` initializes; this only applies to the default Text Field, and is `null` for other variants.
-`leadingIconAriaLabel` | `string` (write-only) | Proxies to the foundation's `setLeadingIconAriaLabel` method.
-`trailingIconAriaLabel` | `string` (write-only) | Proxies to the foundation's `setTrailingIconAriaLabel` method.
-`leadingIconContent` | `string` (write-only) | Proxies to the foundation's `setLeadingIconContent` method.
-`trailingIconContent` | `string` (write-only) | Proxies to the foundation's `setTrailingIconContent` method.
-
-In addition to the above, the following properties proxy to the `input` element's properties of the same name:
-
-* `required`
-* `pattern`
-* `minLength`
-* `maxLength`
-* `min`
-* `max`
-* `step`
-
-Method Signature | Description
+Signature | Description
 --- | ---
+`emit(evtType: string, evtData: T, shouldBubble?: boolean) => void` | Fires a cross-browser-compatible custom event from the component root of the given type, with the given data.
 `focus() => void` | Focuses the `input` or `textarea` element.
+`initialSyncWithDOM() => void` | Initializes the Text Field's internal state based on the environment's state.
 `layout() => void` | Adjusts the dimensions and positions for all sub-elements.
+`listen(evtType: K, handler: SpecificEventListener<K>, options?: AddEventListenerOptions | boolean) => void` | Wrapper method to add an event listener to the component's root element. This is most useful when listening for custom events.
+`unlisten(evtType: K, handler: SpecificEventListener<K>, options?: AddEventListenerOptions | boolean) => void` | Wrapper method to remove an event listener to the component's root element. This is most useful when unlistening for custom events.
 
-## Usage Within Frameworks
+#### Properties
 
-If you are using a JavaScript framework, such as React or Angular, you can create a Text Field for your framework. Depending on your needs, you can use the _Simple Approach: Wrapping MDC Web Vanilla Components_, or the _Advanced Approach: Using Foundations and Adapters_. Please follow the instructions [here](../../docs/integrating-into-frameworks.md).
+Name | Type | Description
+--- | --- | ---
+pattern | `string` | @param Sets the input element's validation pattern.
```

Comment:
Back tick is missing.

Reply:
Are you saying there should be backticks around the method or property name?